File: single-language-analysis/dataset-files/python/test_smell_012.py
import pytest

class TestSoundTest:
    def test_smell_012(self):
        sound = Sound()
        assert 5 == sound.get_octave()

        for i in range(6):
            sound.dec_octave()
        print(sound.get_octave())
        assert 0 == sound.get_octave()

File: single-language-analysis/dataset-files/java/TestSmell012.java
// Original URL: https://github.com/gabrielseibel1/MusicSeibuilder/blob/e931ba43f8edab3ec38006ef8476e77425726952/SoundTest.java#L43-L53

public class TestSmell012 {
    @Test
    private void decOctaveTest() throws Exception{
        Sound sound = new Sound();
        assertEquals(5, sound.getOctave());

        for (int i = 0; i < 6; i++){
            sound.decOctave();
        }
        System.out.println(sound.getOctave());
        assertEquals(0, sound.getOctave());
    }
}
